### ISSUE TITLE
Tiger on VirtualBox 6.1

### ISSUE THREAD

**Issue by fieryo** [ranma42/TigerOnVBox#17]
I tested Tiger (retail) with current VirtualBox 6.1.4 (installed on macOS 10.14) and found these issues with the current README (Thanks for your work @ranma42):

1. Current steps cause "Still waiting for root device." error in almost every second boot. Also, any update of the system (i.e. 10.4.11) that requires restart ends with this error. 
To fix it, before the installation just add the new virtual drive plus optical drive as IDE (ICH6) instead of the default AHCI (SATA). I found this info on VirtualBox forums. With the IDE controller, VM boots every single time without issues. You have to set it before installation, because after that, if you just delete SATA and add IDE controller then VM starts in EFI Shell.

2. Use "USB Tablet" for pointing device. It works properly and your mouse is not locked inside the VM as the current "USB mouse". So please, remove the redundant "USB mouse" setting:
`VBoxManage modifyvm Tiger --mouse usb`

3. Only the following command works to change the VM resolution, instead of the proposed in Post-installation tuning (VBoxInternal2/EfiGopMode n):
`VBoxManage setextradata Tiger VBoxInternal2/EfiGraphicsResolution 1440x900`

**Comment by kencu** [comment]
I got stuck in the EFI shell loop after trying to change the controller type. Using the "Boot Manager" to open the boot.efi on the Tiger Install CD let me get to a menu where I could open "Startup Disk", and once I selected the Tiger HD again everything worked normally again.

**Comment by ranma42** [comment]
Yup, you're right! I just tested your suggestions and they seem to work quite well :)
I am unsure if this also holds for older VirtualBox versions, but I will update the guide for the current one.

The downside of upgrading to 6.1.x seems to be that there has been a regression with SMP support https://github.com/ranma42/TigerOnVBox/issues/18 :(

**Comment by kencu** [comment]
I am no longer able to get more than 1 processor working, though. @ranma42 -- if you can find a way to get > 1 again, that would be much appreciated!

**Comment by JChamberlain28** [comment]
Can confirm the resolution command worked. Been trying to fix my res for a while thanks!
